libraries/AP_BattMonitor/AP_BattMonitor.cpp

```diff
@@ -428,6 +428,29 @@ void AP_BattMonitor::check_failsafes(void)
     }
 }
 
+// returns the total power draw for all batteries
+float AP_BattMonitor::power_watts() const
+{
+    float total = 0;
+    for (uint8_t instance = 0; instance < _num_instances; instance++) {
+        float instance_power;
+        if (power_watts(instance_power, instance)) {
+            total += instance_power;
+        }
+    }
+    return total;
+}
+
+bool AP_BattMonitor::power_watts(float &power, uint8_t instance) const
```

Comment:
Instance number first by convention.

Reply:
By convention ? I just copied the signatures of `AP_BattMonitor::consumed_mah`, `AP_BattMonitor::current_amps`, ...